sample_csvs/sec109_closed_school.csv

```diff
@@ -0,0 +1,8 @@
+facility_code,school_name,closure109
```

Comment:
is this data from production?

Reply:
No I had to make it up to test it.

Reply:
Remove data from file, just leave headers.  When production data is available this sample csv will be updated to reflect production